LICENSES/CC0-1.0.txt

```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
+
+CC0 1.0 Universal
```

Comment:
what's the problem with the public domain license in the existing LICENSE file?
I know that public domain is contentious (why?) but it seems like a good license for this sort of thing.

Reply:
Public Domain is legally problematic because it's not a license, it's you giving up your rights.

That's not allowed in a bunch of jurisdictions (like the EU), which is why CC0 exists.